Lib/test/test_perf_profiler.py

```diff
@@ -0,0 +1,153 @@
+import unittest
+import subprocess
+import re
+import sys
+import sysconfig
+import os
+from test import support
+from test.support.script_helper import make_script
+from test.support.os_helper import temp_dir
+
+
+if not support.has_subprocess_support:
+    raise unittest.SkipTest("test module requires subprocess")
+
+
+def is_unwinding_realiable():
```

Comment:
```suggestion
def is_unwinding_reliable():
```
Is it a typo here? (unnecessary letter "a")